### ISSUE TITLE
more of a question to the developers

### ISSUE THREAD

**Issue by IIIdefconIII** [yuliskov/SmartTubeNext#887]
Im sorry instead of going the long Telegram way i would ask if you can change the app its name?
![afbeelding](https://user-images.githubusercontent.com/9917659/132974045-48970713-35b9-4afd-8079-2357bce90f61.png)

As its much nicer in my HomeAssitant

com.teamsmart.videomanager.tv to SmarTubeNext

**Comment by yuliskov** [comment]
Sure. Thanks.

**Comment by IIIdefconIII** [comment]
> Sure. Thanks.

Wow cool, thanks!

**Comment by yuliskov** [comment]
@IIIdefconIII one more question. What's home assistant? Are you taking about this project?
https://www.home-assistant.io

**Comment by IIIdefconIII** [comment]
@yuliskov yes indeed. Its a home automation software wivh has alot of smart devoces intergrated.

**Comment by IIIdefconIII** [comment]
In which update we can supsect this? Wpuld be very cool

**Comment by IIIdefconIII** [comment]
> @IIIdefconIII one more question. What's home assistant? Are you taking about this project?
> https://www.home-assistant.io

Thanks for this but i already found it eith a simple netstay

**Comment by IIIdefconIII** [comment]
@yuliskov any idea when this change could happen?
